src/app/components/ATIAnalytics/canonical/sendBeaconOperaMini.js

```diff
@@ -0,0 +1,35 @@
+import isOperaProxy from '#app/lib/utilities/isOperaProxy';
```

Comment:
Technically we could make this file Typescript, but we would need to use the [Compiler API](https://github.com/Microsoft/TypeScript/wiki/Using-the-Compiler-API), to be able to inline properly